src/cargo/core/compiler/build_runner/mod.rs

```diff
@@ -256,12 +256,9 @@ impl<'a, 'gctx> BuildRunner<'a, 'gctx> {
                             args.push(cfg.into());
                         }
 
-                        if !output.check_cfgs.is_empty() {
-                            args.push("-Zunstable-options".into());
```

Comment:
rustc needs to be stabilized first

(yes, you mention this and this is a draft but I wanted an explicit review status also)